src/plotman/plotman.py

```diff
@@ -95,8 +95,22 @@ def parse_args(self) -> typing.Any:
         p_analyze.add_argument('--bybitfield',
                 action='store_true',
                 help='slice by bitfield/non-bitfield sorting')
-        p_analyze.add_argument('logfile', type=str, nargs='+',
+        p_analyze.add_argument('--logfile', type=str, nargs='+', default=None,
                 help='logfile(s) to analyze')
+        p_analyze.add_argument('--logdir', type=str, default=None,
+                help='directory containing multiple logfiles to analyze')
+        p_analyze.add_argument('--figfile', type=str, default=None,
+                help='figure to be created if logdir is passed')
```

Comment:
I think that the changes to analyze are not related to the graphing addition anymore and can be removed?